app/views/declaration/not_declarant.scala.html

```diff
@@ -23,15 +23,18 @@
 @this(
         govukLayout: gdsMainTemplate,
         pageTitle: pageTitle,
-        formHelper: FormWithCSRF
+        formHelper: FormWithCSRF,
+        link: link
 )
 
 @()(implicit request: Request[_], messages: Messages)
 
 @paragraph1Link = {<a href=@controllers.routes.RootController.displayPage.url>@messages("notDeclarent.paragraph1Link")</a>}
 @paragraph2Link = {<a href="https://www.gov.uk/guidance/dispatching-your-goods-within-the-eu">@messages("notDeclarent.paragraph2Link")</a>}
 @supportPhone = {<b>0300 200 3700</b>}
-@enquiriesLink = {<a href="https://www.gov.uk/government/organisations/hm-revenue-customs/contact/customs-international-trade-and-excise-enquiries" target="_blank" rel="noopener noreferrer">@messages("general.inquiries.help.link")</a>}
+@enquiriesLink = {
+  @link(text = messages("general.inquiries.help.link"), call = Call("GET", "https://www.gov.uk/government/organisations/hm-revenue-customs/contact/customs-international-trade-and-excise-enquiries"), target = "_blank")
```

Comment:
It's not that I'm against to have the url hardcoded here, but it seems to me at odds with the logic we usually follow... the urls are in `application.conf`. And this is the same for all the other urls I'm seeing hardcoded in the templates in this PR (I do not want to repeat the same comment)